lib/src/provider.dart

```diff
@@ -202,32 +215,16 @@ class StatefulProvider<T> extends StatefulWidget
   /// Allows to specify parameters to [StatefulProvider]
   StatefulProvider({
     Key key,
-    @required this.valueBuilder,
     this.child,
+    @required this.valueBuilder,
```

Comment:
[nit] I think it's good style to start with `@required` paramaters.

Reply:
done!